src/transformers/benchmark/benchmark_args_utils.py

```diff
@@ -61,6 +61,12 @@ class BenchmarkArguments:
     save_to_csv: bool = field(default=False, metadata={"help": "Save result to a CSV file"})
     log_print: bool = field(default=False, metadata={"help": "Save all print statements in a log file"})
     no_env_print: bool = field(default=False, metadata={"help": "Don't print environment information"})
+    with_lm_head: bool = field(
```

Comment:
allow the possibility to switch between model + lm_head and without

Reply:
Cool, I think this makes sense.